repo_tests/check_for_task_failures.py

```diff
@@ -19,11 +19,6 @@ def main(github_env_path, repo_test_name, repo_test_description):
   print(open(".trunk/trunk.yaml").read())
   print("::endgroup::")
 
-  with open(github_env_path, 'a') as env:
-    env.write('landing_state_artifact_name=')
-    env.write(f'{repo_test_name.replace("/", " ")} ({repo_test_description})')
-    env.write('\n')
-
```

Comment:
unsure what writing this env var does, but it was erroring in the docker case and the env var doesn't seem to get used, so I removed this